src/components/sbb-link/sbb-link.scss

```diff
@@ -81,15 +80,15 @@
   }
 }
 
-.sbb-link__icon {
+sbb-icon {
   width: var(--sbb-size-icon-ui-small);
   height: var(--sbb-size-icon-ui-small);
 }
 
-.sbb-link--icon-placement-end .sbb-link__icon {
+.sbb-link--icon-placement-end sbb-icon {
   order: 2;
 }
 
-.sbb-link[dir='rtl'].sbb-link--icon-flip .sbb-link__icon {
+.sbb-link.sbb-link--icon-flip sbb-icon {
```

Comment:
As reading the iconFlip docs, I think it should only be flipped on rtl mode. Why did you remove the `[dir='rtl']` part?

Reply:
[dir='rtl'] is used to set the reading direction of text and should not be connected to to and icon that has no contextual  meaning.

Reply:
But for what do we need the icon flip property then? Or you have at least to update the docs there. 

Your changes made that you now can flip an icon regardless of the writing direction.

Reply:
I did not write the docs. They do not make sense to me. If you are using Icons that has a contextual meaning it should get a description somehow via an attribute (alt/aria-label or so). I dont get the idea behind that. But to flip the orientation of the icon depending on the position sounds logical to me.